docs/tutorials/install.rst

```diff
@@ -114,6 +114,44 @@ with a *PostgreSQL* container.
     wget https://raw.githubusercontent.com/Kinto/kinto/master/docker-compose.yml
     sudo docker-compose up
 
+If you want to change the settings, you need to mount a custom settings file
+into the *Kinto* container. Hopefully Docker Compose let you do that the exact
+same way Docker does (assuming you have created the config file ``./config/kinto.ini``):
+
+::
```

Comment:
I do not know rST very well, so I *cargo cult*-ed the block code syntax. Is it wrong?